model/src/main/scala/hmda/model/census/CBSAMetroMicroLookup.scala

```diff
@@ -0,0 +1,56 @@
+package hmda.model.census
+
+import java.io.File
+
+import scala.io.Source
+
+object CBSAMetroMicroLookup {
+  val values: Seq[CBSAMetroMicro] = {
+    val lines = Source.fromFile(new File("model/src/main/resources/tl_2013_us_cbsa.csv")).getLines
+    lines.map { line =>
+      val values = line.split('|').map(_.trim)
+      val combinedCode = values(0)
+      val metroOrMicroCode = values(1)
+      val geoId = values(2)
+      val name = values(3)
+      val legalStatname = values(4)
+      val legalStatMetroMicro = values(5)
+      val metroMicro = values(6).toInt
+      val metroMicroStatArea = values(7)
+      val areaLand = values(8)
+      val areaWater = values(9)
+      val internalPointLat = values(10)
+      val internalPointLon = values(11)
+
+      CBSAMetroMicro(
+        combinedCode,
+        metroOrMicroCode,
+        geoId,
+        name,
+        legalStatname,
```

Comment:
Should the 'n' should be capitalized, to make this `legalStatName`? (I'm unsure, since there are lots of complexities of geocoding that I don't understand yet. But using words for these variables instead of acronyms is really helpful!)